src/utils/annotation.ts

```diff
@@ -91,17 +91,27 @@ export function simpleCentroid(coordinates: IGeoJSPoint[]): IGeoJSPoint {
   if (coordinates.length === 1) {
     return coordinates[0];
   }
-  const sums: IGeoJSPoint = { x: 0, y: 0, z: 0 };
+  const sums = { x: 0, y: 0, z: 0 };
+  const allZ = !coordinates.some(coord => coord.z === undefined);
   coordinates.forEach(({ x, y, z }) => {
     sums.x += x;
     sums.y += y;
-    sums.z += z;
+    if (z !== undefined) {
```

Comment:
test instead `allZ` ?

Reply:
It would work and that's what I initially did, but typescript then tells me for z that `Object is possibly 'undefined'.ts (2532)`
I don't know how to tell typescript that it is not undefined

Reply:
I believe there are special comments that force typescript to ignore some cases, no ?

Reply:
Indeed, I found `// @ts-ignore` comment to ignore a line